### DISCUSSION TITLE
Start db server for multi-threaded compiles

### DISCUSSION THREAD

**Discussion by ntreeby** [Riverside-Software/pct#531] [Ideas]
If we create a local empty db from a schema for our compiles, we can only run a compile single-threaded since we have no obvious way to start the db in multi-user mode.

It would be helpful to have a PCTProserve (or similar) task that will start the db server so it can be used by a multi-threaded PCTCompile.

**Comment by stefandrissen** [comment]
No need for serving databases, just connect you database read-only with -RO.

**Reply by gquerret** [reply]
Confirmed.
